docs/content/components/layout/columns/fit-width-columns.demo.tsx

```diff
@@ -0,0 +1,10 @@
+import { Columns, Switch, TextField } from '@marigold/components';
+
+export default () => (
+  <Columns space={2} columns={[8, 'fit']} collapseAt="25em">
+    <TextField label="Setting 1" />
+    <div className="align-center flex">
```

Comment:
The alignment doesn't seems to work.

Reply:
it is aligned, it only looks weird because of the label from the field... :/ Do you have an idea to solve this?